src/researchhub_document/views/researchhub_unified_document_views.py

```diff
@@ -672,34 +634,41 @@ def get_unified_documents(self, request):
         hub_id = query_params.get("hub_id", 0) or 0
         page_number = int(query_params.get("page", 1))
 
-        cache_hit = self._get_unified_document_cache_hit(
-            document_request_type,
-            filtering,
-            hub_id,
-            page_number,
-            time_scope,
-        )
-
-        if cache_hit and page_number == 1:
-            cache_hit = self._cache_hit_with_latest_metadata(cache_hit)
-            return Response(cache_hit)
-        elif not cache_hit and page_number == 1:
-            reset_unified_document_cache(
-                document_type=[document_request_type],
-                filters=[filtering],
-                date_ranges=[time_scope],
-            )
-
-        documents = self.get_filtered_queryset().prefetch_related("fundraises")
+        # cache_hit = self._get_unified_document_cache_hit(
```

Comment:
This will be commented out before merging

Reply:
Do you mean the code will be put back in? Otherwise better to remove it.